cli/create.py

```diff
@@ -295,7 +295,7 @@ def create(
     )
 
     # save horcrux private key to the keystore
-    keystore = create_keystore(
+    keystore = HorcruxPbkdf2Keystore.from_private_key(
```

Comment:
The `HorcruxPbkdf2Keystore` is used for encrypting the private key and storing it in the keystore file. I suggest renaming `from_private_key` to something else like `create`?

Reply:
Renamed in 06ac5c4.